charts/quickwit/values.yaml

```diff
@@ -239,6 +245,10 @@ control_plane:
 
   podAnnotations: {}
 
+  serviceAnnotations: {}
+
+  serviceType: {}
```

Comment:
Same comment, we can have something like:

```yaml
control_plane:
   service:
     # type: ClusterIP
     annotations: {}
```

Instead of `serviceAnnotations` and `serviceType`